docs/book/event-manager.md

```diff
@@ -53,13 +53,13 @@ Handled event "do", with parameters {"foo":"bar","baz":"bat"}
 >
 > Throughout this tutorial, we use closures as listeners. However, any valid PHP
 > callback can be attached as a listeners: PHP function names, static class
-> methods, object instance methods, functors, or closures. We use closures
+> methods, object instance methods, function, or closures. We use closures
 > within this post for illustration only.
 
 ### Event instances
 
 `trigger()` is useful as it will create a `Laminas\EventManager\Event` instance for
-you. You may want to create such an instance manually; for instance, you may
+you. You may want to create such an instance manually for instance, you may
```

Comment:
Thank you for your time and this contribution! 👍 

The whole sentence is difficult to read and understanding the content is therefore also difficult. There are too many repetitions of the same words. A list of the different possibilities would improve readability.

Reply:
So let me revert with the original one.

Reply:
I'm sorry, I think my wording was inadequate. With "a list of the different possibilities" I mean an enumeration / bullet list.